06 piškvorky/tick_tack_toe_with_tests/test_TickTackToe.py

```diff
@@ -0,0 +1,72 @@
+from piskvorky import move
+from ai import pc_move
+import time
+
+
+player_symbol = "X"
+pc_symbol = "O"
+play_area = 20 * "-"
+
+
+def test_move():
+    symbol = 'X'
+    assert move(play_area, 19, symbol) == "-------------------X"
+    assert move(play_area, 0, symbol) == "X-------------------"
+
+
+def test_pc_move():
+    area_test = pc_move(20 * "-")
+    area_test_next = pc_move("--X-----------O-----")
+    assert len(area_test) == 20
+    assert area_test.count('O') == 1
+    assert area_test.count('-') == 19
+    assert len(area_test_next) == 20
+    assert area_test_next.count('O') == 2
+    assert area_test_next.count('-') == 17
+    assert area_test_next.count('X') == 1
+
+
+def score(area):
+    # function get list(area) and return result in accordance with area
+    if (3 * player_symbol) in area:
+        return player_symbol
+    elif (3 * pc_symbol) in area:
+        return pc_symbol
+    elif '-' not in area:
+        return 0
+    else:
+        return "-"
+
+
+def tick_tack_toe_test(play_area):  # all together in game
+
+    while True:
+        play_area = player_move(play_area)
+        print(play_area, "hráč")
+        score(play_area)
```

Comment:
Nie je potreba to volať tu a potom aj na nasledujúcom riadku. To isté platí pre riadky 52+53. Stačí zachovať to volanie v podmienke, toto (riadky 46 a 52)  na nič nevyužívaš :)